nss-day-cohort-48/the-worst-site-ever-fandango
# added satisfied customers section

added awesome satisfied customers section

##steps to test

1. pull and serve
2. check that the satisfied customers section is at the bottom of the home page
